test/runtests.jl

```diff
@@ -10,19 +10,19 @@ solvers = [
            ODE.ode2_midpoint,
            ODE.ode2_heun,
            ODE.ode4,
-           ODE.ode4ms,
-           ODE.ode5ms,
+           # Ode.ode4ms,
```

Comment:
Yes, I think it's fine to leave some old-style methods and update them another day.  In fact, maybe it makes sense to keep some simple fixed-step methods as they are, to retain some easy to understand solvers.